src/sql/workbench/parts/jobManagement/electron-browser/jobManagementView.ts

```diff
@@ -72,20 +73,26 @@ export abstract class JobManagementView extends TabChild implements AfterContent
 
 	abstract onFirstVisible();
 
-	protected openContextMenu(event): void {
-		let rowIndex = event.cell.row;
+	protected openContextMenu(event: ITableMouseEvent): void {
+		const rowIndex = event.cell.row;
 
-		let targetObject = this.getCurrentTableObject(rowIndex);
-		let actions = this.getTableActions(targetObject);
+		const targetObject = this.getCurrentTableObject(rowIndex);
+		const actions = this.getTableActions(targetObject);
 		if (actions) {
-			let ownerUri: string = this._commonService.connectionManagementService.connectionInfo.ownerUri;
-			let actionContext: IJobActionInfo = {
+			const ownerUri: string = this._commonService.connectionManagementService.connectionInfo.ownerUri;
+			const actionContext: IJobActionInfo = {
 				ownerUri: ownerUri,
 				targetObject: targetObject,
 				component: this
 			};
 
-			let anchor = { x: event.pageX + 1, y: event.pageY };
+			let anchor: { x: number, y: number };
+			if (event.anchor instanceof HTMLElement) {
+				anchor = { x: event.anchor.getBoundingClientRect().left + 1, y: event.anchor.getBoundingClientRect().top };
```

Comment:
I don't think you need to do this conditional, the anchor type on the event is designed to be passed directly as the anchor to the contextmenuservice, which accepts a object or an htmlelement as a anchor type.

Reply:
Sometimes the answer is right in front of your face, thanks!